packages/core/src/actors/observable.ts

```diff
@@ -209,6 +209,9 @@ export function fromObservable<TContext, TInput>(
  * ```
  */
 
+const errorEventType = '$$xstate.error';
+const completeEventType = '$$xstate.complete';
```

Comment:
We can also deduplicate it with the same constants that are defined within `fromObservable`. I would also drop `$$` prefix from all of them